app/assets/stylesheets/custom.css.scss

```diff
@@ -105,6 +105,15 @@ li { list-style: none; }
   background-color: $site-brown;
 }
 
+#sign-in { margin: 50px; }
+
+#supply-hist {
+  margin-top: 0px;
+}
+
+#profile-table{
```

Comment:
Avoid using id selectors<br>Opening curly brace `{` should be preceded by one space